src/components/App/App.component.jsx

```diff
@@ -1,57 +1,61 @@
-import React, { useLayoutEffect } from 'react';
-import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
-import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import React from 'react';
+import Navbar from '../Navbar/Navbar.component'
+import ListVideos from '../ListVideos/ListVideos.component';
+import videos from '../../videos.js';
```

Comment:
try to use the same format for the imports, is not really necessary to use `.js` on the path, is already added per default